packages/kit/src/runtime/shared-server.js

```diff
@@ -4,8 +4,8 @@ export let private_env = {};
 /** @type {Record<string, string>} */
 export let public_env = {};
 
-/** @param {string} stack */
-export let fix_stack_trace = (stack) => stack;
+/** @param {any} _error */
+export let fix_stack_trace = (_error) => '';
```

Comment:
This function likely isn't ever reached in this state, but just to be safe and ensure we're not swallowing anything, should it be:

```suggestion
export let fix_stack_trace = (_error) => _error?.stack;
```

Reply:
Updated it with the suggestion (but removed the _ prefix since the arg is used)